coopterrones/rancid-tomatoes
# Feature/iteration1

# Pull Request


#### What's this PR do?
1. Adds all immediately necessary files to project for iteration 1 components and styling
2. Adds sample `movieData`
3. Display all movies on main page 
4. Clean up component implementation code
5. Adds SCSS to `<MovieCard/>` components

#### Where should the reviewer start?
-`App.js` line 6 -- add class component for App and initial state
-`App.js` line 14 -- add render method 
-`MovieCard.js` line 4 -- add movie card functional component 
-`Movies.js` line 5 -- add movies functional component
-`movieData.js` line 1 -- adds all sample movie data
-`MovieCard.scc` line 1 -- adds MovieCard styling to main page

#### How should this be manually tested?
-run `npm start` and check that all movie cards are loading with title, average rating, and release date on main page load. 
-check for errors in console
-cross reference with sample data properties for correct display of info for each movie

#### Any background context you want to provide?
-Need media queries for macbook screen since we were both dummies looking on our monitors lol
-Media queries for mobile and most likely touching up with tablet
-Additional styling can be sorted out for overall theme.. LETS DO NEUMORPHISM :)

#### Screenshots (if appropriate):
![mainPageView](https://user-images.githubusercontent.com/67164959/100810738-55d62100-33f6-11eb-8586-df56efcc87f8.png)


#### Questions
-Are you ready to friggn rock ..
